### ISSUE TITLE
Error generating thumbnails or encoding when file is uploaded to Amazon S3

### ISSUE THREAD

**Issue by daoom** [kylegilman/video-embed-thumbnail-generator#367]
I am using the WP Offload S3 plugin tp host my video files on AWS. 

When I try to generate thumbnails to encode the file, I get the following error message: 

> Can't open movie file.
> libpostproc 54. 0.100 / 54. 0.100
> s3://{bucketname}/wp-content/uploads/2016/10/18003054/Juice-Box.mp4: Protocol not found

The actual path is
"http://s3.amazonaws.com/{bucketname}/wp-content/uploads/2016/10/18003054/Juice-Box.mp4"

Where "{bucketname}" is the name of storage bucket I created,

**Comment by kylegilman** [comment]
Sorry I didn't respond to this sooner. Since WP Offload S3 is a commercial plugin, I don't have access to it and can't troubleshoot it. It seems like it's filtering the results of `wp_get_attachment_url` in a way that returns the s3:// protocol instead of http:// but that's just guessing.

**Comment by dmkjr** [comment]
Hello - Was curious if there was any movement on this?  I see your response about it being a commercial plugin.  I will report to them as well, but just checking on any movement.

**Comment by kylegilman** [comment]
No, no movement. I continue to have no access to this commercial plugin.

**Comment by dmkjr** [comment]
I could provide a copy of it for integration if you are interested.

**Comment by dmkjr** [comment]
@kylegilman The developers of WP Offload have added compatibility with your plugin to their road map.  Just an FYI.
